bot/controllers/twitchLive.js

```diff
@@ -0,0 +1,32 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+// const util = require('apex-util');
+
+class StreamerLiveController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command(
+        '!twitchLive',
```

Comment:
Since your goal is to make the StreamerLiveController generic, consider changing the command too!live and add a service attribute, for example !live twitch myname mygame.

Reply:
I am going to implement this method making one command good for twitch or mixer